### ISSUE TITLE
question about loss function design

### ISSUE THREAD

**Issue by EcustBoy** [autonomousvision/plant#9]
Hi~ RenzKa, thanks for your interesting work !  I noticed that the loss function design in the paper is relatively simple, in which there is only imitation loss， so **I think it's an open-loop training paradigm.**  

here is my two question： 

(1) **why can the open-loop model be directly used in closed-loop simulation environment**?   (In my opinion, this can lead to errors accumulating and simulation collapse, like what's said in the early method ChauffeurNet [https://arxiv.org/abs/1812.03079](url))

(2) besides,  I would like to know if you have tried to **add collision losses** between ego car and other cars for comparative experiments in addition to pure imitating losses?  (Accoarding to my experience in nuscenes dataset, only using imitation loss may cause unreasonable collision behavior)

I'll thanks much for your reply~

**Comment by kashyap7x** [comment]
Hi @EcustBoy, thanks for your interest in our work!

(1) All of the SOTA methods on CARLA currently perform only open-loop training (without DAgger, collision losses, etc). Our hypothesis on why this works is that the GRU decoder module used in all these works (as well as PlanT) have a strong bias to extrapolate waypoints towards the provided GPS target point input, which in our simulator is always well-localized to the center of the lane. Most of the minor drift is naturally corrected this way.

(2) We haven't tried to add collision losses. I have observed weird collision errors in nuPlan, mostly due to causal confusion, where the agent simply extrapolates its trajectory history as a future plan, ignoring other vehicles. In PlanT, we do not provide the model with ego-history information, which helps. Maybe you can try this in your setting.
